GANDLF/grad_clipping/adaptive_gradient_clipping.py

```diff
@@ -28,7 +28,12 @@ def unitwise_norm(x, norm_type=2.0):
         return x.norm(norm_type, dim=tuple(range(1, x.ndim)), keepdim=True)
 
 
-def adaptive_gradient_clip_(parameters, clip_factor=0.01, eps=1e-3, norm_type=2.0):
+def adaptive_gradient_clip_(
```

Comment:
If indeed as the docstring says, parameters can be either list or torch tensor, the annotation should be changed to include that. Also, as I mentioned in the comment below, I believe the list should be annotated with the typing List[object_types] format

Reply:
Fixed.